extensions/wikia/Captcha/Captcha.setup.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/**
+ * @file
+ * @ingroup Extensions
+ */
+
+if ( !defined( 'MEDIAWIKI' ) ) {
```

Comment:
I think this is standard boilerplate on 3rd party extensions.  It's meant for:

  http://www.mediawiki.org/wiki/Register_globals#Protecting_MediaWiki_from_Register_Globals

so we can probably get rid of it